sql/size_utils.sql

```diff
@@ -162,50 +138,52 @@ $BODY$;
 -- total_bytes        - Total disk space used by the specified table, including all indexes and TOAST data
 
 CREATE OR REPLACE FUNCTION hypertable_detailed_size(
-    hypertable              REGCLASS
-)
+    hypertable              REGCLASS)
 RETURNS TABLE (table_bytes BIGINT,
                index_bytes BIGINT,
                toast_bytes BIGINT,
                total_bytes BIGINT,
-               node_name   NAME
-               ) 
+               node_name   NAME)
 LANGUAGE PLPGSQL VOLATILE STRICT AS
 $BODY$
 DECLARE
-        table_name       NAME;
-        schema_name      NAME;
-        is_distributed   BOOL;
+        table_name       NAME = NULL;
+        schema_name      NAME = NULL;
+        is_distributed   BOOL = FALSE;
 BEGIN
         SELECT relname, nspname, replication_factor > 0
-        INTO STRICT table_name, schema_name, is_distributed
+        INTO table_name, schema_name, is_distributed
         FROM pg_class c
         INNER JOIN pg_namespace n ON (n.OID = c.relnamespace)
         INNER JOIN _timescaledb_catalog.hypertable ht ON (ht.schema_name = n.nspname AND ht.table_name = c.relname)
         WHERE c.OID = hypertable;
 
+		IF table_name IS NULL THEN
+		    RAISE EXCEPTION 'relation "%" is not a hypertable', hypertable::text
+			      USING HINT = 'Use pg_relation_size() to get the size of other relations.',
+				  ERRCODE = 'invalid_parameter_value';
+		END IF;
+
         CASE WHEN is_distributed THEN
-            RETURN QUERY SELECT * FROM _timescaledb_internal.hypertable_remote_size(schema_name, table_name);
```

Comment:
Suggest to return the size of the access node hypertable with `NULL` as the name, similar to how it's done for the non-distributed case.

Reply:
I plan to do that, although I am not sure returning a NULL name is the best approach. We could just return "access node", although that has other issues if you, e.g., give a data node that name as well (unlikely).

Reply:
Well... my thinking was that since you use NULL for the local query, it makes sense to use NULL for the local part of the remote query. If you use a concrete name for the local part in the remote query, you should probably do the same for the fully local query, for consistency.

Reply:
Note also that I think it makes sense to use `NULL` for the name since the access node does not have a name.

Reply:
Now returns NULL.